Reference/Mapping/index.md

```diff
@@ -0,0 +1,190 @@
+---
+versionFrom: 8.1.0
+---
+
+# UmbracoMapper
+
+:::tip
+UmbracoMapper replaced AutoMapper which was an external dependency. AutoMapper builds the mapping code dynamically, based upon mapping profiles, which are defined as C# expressions. UmbracoMapper relies on static code, i.e. mappings need to be hand-written.
+
+This is not to be confused with the [package by Andy Butland](https://our.umbraco.com/packages/developer-tools/umbraco-mapper)
+:::
+
+## Accessing the UmbracoMapper
+
+The UmbracoMapper is registered with Dependency Injection (DI). It can therefore be injected in constructors, wherever DI is used. Alternatively, it is also exposed via `Current.Mapper`.
```

Comment:
injected into constructors, of controllers, custom classes etc, wherever DI is used. Alternatively, it is also exposed via the Current service locator: 'Current.Mapper'